### ISSUE TITLE
Python: Document default settings

### ISSUE THREAD

**Issue by geigerj** [googleapis/toolkit#197]
The default settings (as stored in the client config JSON) should also appear in the documentation.

Proposal: use the GAPIC config to generate a RST file that Sphinx can incorporate into the generated docs.

**Comment by bjwatson** [comment]
Copied from https://github.com/googleapis/gax-python/issues/106:

During the usability study, participants intuitively understood the role of the CallOptions parameter, and its implications on their API requests.

However, for those that investigated further, it was not clear what the default CallOptions setting were, and thus it was difficult to predict the behavior of their requests without first manually setting the options.

Recommendation: Ensure that default CallOptions are accurately and clearly documented. If default options differ by API request, consider adding the defaults to each method’s reference documentation, so user can accurately predict the behavior of their request.

/cc @rok987 
/cc @andrewmacvean
/cc @tbetbetbe

**Comment by eoogbe** [comment]
@geigerj Is this still relevant?

**Comment by geigerj** [comment]
Yes, AFAIK we still don't document the default method configuration in any of the GAPIC docs.

**Comment by geigerj** [comment]
Unassigning to go back on the backlog -- I'm not going to have bandwidth for this in the near future.
